### DISCUSSION TITLE
AWS QLDB/QLDB Session

### DISCUSSION THREAD

**Discussion by 80kk** [mockoon/mockoon#949] [General]
Hi,
Do I need both QLDB and QLDB Session to mock AWS QLDB service properly or QLDB Session API is enough?

**Comment by 255kb** [comment]
Hi, it depends on which API endpoints you are using. I cannot really help with this specific API as I don't know how it works, but if you are using endpoints of both QLDB and QLDB Session, you may want to mock them all. 
But you don't necessarily have to mock everything. Maybe it's ok to call the real APIs in your app for some endpoints and use the [proxy mode](https://mockoon.com/tutorials/partial-mocking-proxy/) to mock only the most complex endpoints, or the paid ones.
